tests/drivers/build_all/bbram/testcase.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 TOKITA hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+tests:
```

Comment:
Please add a `common:` section for the common fields.

Reply:
Fixed it.